src/cloud/docker/docker-extend.md

```diff
@@ -27,85 +27,129 @@ You can use the built-in extension mechanism of Docker to specify [multiple comp
 
 ## Specify Docker build sources
 
-To test changes to images or make more extensive changes to the containers, you must build them from source. You can do this by
-by adding the `build:context` configuration to the `docker-compose.override.yml` file.
+To test changes to images or make more extensive changes to the containers, you must build them from source. You can do this by adding the `build:context` configuration to the `docker-compose.override.yml` file.
 
 The following example defines the build context for the Web container. You can use the same technique to build from any of the images in  the `vendor/magento/magento-cloud-docker` directory or any other Docker image, including local images that are resourced outside the project.
 
-```yaml
-version: '2.1'
-services:
-   web:
-    build:
-     context: ./vendor/magento/magento-cloud-docker/images/nginx/1.9/
-```
+  ```yaml
+  version: '2.1'
+  services:
+    web:
+      build:
+        context: ./vendor/magento/magento-cloud-docker/images/nginx/1.9/
+  ```
 
 To update the container configuration and test iteratively, use the `--force-recreate` option to refresh the container build:
 
-```bash
-docker-compose up -d --force-recreate --build
-```
+  ```bash
+  docker-compose up -d --force-recreate --build
+  ```
 
 ## Add a new version of existing service
 
-In {{site.data.var.mcd}} package the available [service versions] are determined by the Docker service images configured in the {{site.data.var.mcd}} `images` directory. You can add a new service version by creating a directory for the version and adding a `Dockerfile` and other files to configure the new version.
+In the `{{site.data.var.mcd-package}}` package, the available [service versions] are determined by the Docker service images configured in the `images` directory. You add a new service version by creating a directory for the version and adding a `Dockerfile` and other files to configure the new version.
 
 {:.procedure}
 To add a new service version using a `Dockerfile`:
 
-1. Clone the `{{site.data.var.mcd}}` project to your local environment if necessary.
+1. Clone the `{{site.data.var.mcd-package}}` project to your local environment if necessary.
 
 1. On the command line, change to the directory that contains the existing service version configurations.
 
    ```bash
-      cd magento-cloud-docker/images/<service-name>
+   cd magento-cloud-docker/images/<service-name>
    ```
 
 1. Create a directory for the new version.
 
-1. In the new directory, add the `Dockerfile` that contains the image configuration details for the new version. Use the `Dockerfile` for an existing version as a template and add any other required configuration such as supported plugins.
+1. Using the `Dockerfile` for an existing version as a template, create the `Dockerfile` for the new version in the version directory.  Make sure that the new file contains any additional required configuration details such as supported plugins.
```

Comment:
The first sentence is confusing. I would stick with a very simple statement, and then add details in the next sentence or subsection.
Also, you never tell them to change to the new directory. If you carry them into a space, you don't have to explain where to create something. They are already there.

Perhaps something like:
```suggestion
1. Change to the new directory.

1. Create a `Dockerfile` with any additional configuration details for the new version, such as supported plugins. You can use the `Dockerfile` from the previous version as a template.
```